src/components/AdvertiserName/AdvertiserName.tsx

```diff
@@ -25,7 +25,7 @@ const AdvertiserName = ({ advertiserStats, onClickBlocked }: TAdvertiserNameProp
         <div className='advertiser-name' data-testid='dt_advertiser_name'>
             <UserAvatar nickname={name ?? ''} size={isDesktop ? 64 : 42} textSize='lg' />
             <div className='advertiser-name__details'>
-                <div className='flex items-center gap-3'>
+                <div className='flex items-center gap-3' data-testid='dt_advertiser-name_nickname'>
```

Comment:
We need to use snake_case as our naming convention for data-testids.
```suggestion
                <div className='flex items-center gap-3' data-testid='dt_advertiser_name_nickname'>
```